### PR TITLE
docs: update browser ipns publish example

### PR DESCRIPTION
**Pull request by MisterY**
Applying all the magic from #3584,
updating the dependencies,
adding tips (address example) for working with a local node

### CODE REVIEW

**Comment by lidel** [comment]
Blocked until https://github.com/ipfs/js-ipfs/issues/2945 is done :)

File: examples/browser-ipns-publish/index.html [hunk 1]
```diff
@@ -137,6 +137,16 @@ <h1 class="aqua fw2 montserrat dib ma0 pv2 ph1 v-mid fr f3 lh-copy">
       >
         Connect
       </button>
+      <!-- For localhost:
+        placeholder=""
+      -->
```
Comment (vasco-santos) [inline examples/browser-ipns-publish/index.html:142]:
```suggestion
```

File: examples/browser-ipns-publish/index.html [hunk 2]
```diff
@@ -137,6 +137,16 @@ <h1 class="aqua fw2 montserrat dib ma0 pv2 ph1 v-mid fr f3 lh-copy">
...
+      <div
+        class="custom-block callout"
+        style="font-size: 0.75em; width: 75%; margin: 1em;"
+      >
+        For connecting to a local go-ipfs node, you may need to use
```
Comment (vasco-santos) [inline examples/browser-ipns-publish/index.html:147]:
```suggestion
        For connecting to a local go-ipfs node, you can check its listening address similar to:
```

**Review by vasco-santos — CHANGES_REQUESTED** [review]
Thanks for the PR @MisterY  and apologies for a late review

I have two requests here:

- can you rebase this PR to fix the conflicts?
- can you update the Readme? The addresses in https://github.com/ipfs/js-ipfs/tree/master/examples/browser-ipns-publish#3-open-the-demo-in-a-browser-and-connect-to-the-go-node should be the websockets addrs as you added in the html file, not TCP as browsers do not speak tcp

**Comment by BigLep** [comment]
@Gozala : are you able to check to see if this can be merged now?

**Comment by vasco-santos** [comment]
@MisterY can you please rebase the PR with master as this seems to have a conflict now?

**Comment by MisterY** [comment]
Updated!

**Review by vasco-santos — APPROVED** [review]
LGTM thanks @MisterY

**Comment by oliveriosousa** [comment]
Hi @MisterY, 

Thanks for your improvements, I was having problems connecting using web sockets and this did the trick.

I just want to add one thing, it might be needed to change one line of code:

https://github.com/MisterY/js-ipfs/blob/c70422c07f67401980f10488cd2e8b765b2885b9/examples/browser-ipns-publish/index.js#L58-L73

From this,
`ipfsAPI = IpfsHttpClient(url)`

To:
`ipfsAPI = IpfsHttpClient.create(url)`

May be related to an update of `ipfs-http-client`

**Review by achingbrain — APPROVED** [review]
LGTM, just waiting on CI

**Pull request merged**